lib/ruby_lsp/requests/hover.rb

```diff
@@ -164,7 +164,7 @@ def generate_gem_hover(node)
         end
 
         # Remove leading whitespace if a heredoc was used for the summary or description
-        info = info.gsub(/^ +/, "") if info
+        info = info&.gsub(/^ +/, "")
```

Comment:
Can `info` ever be nil here? The `find` above suggests it can't.

Reply:
Added a `T.let` to clarify the expected type for Sorbet and removed the safe navigation.